### DISCUSSION TITLE
Question about the velocity input of airsim api moveToPositionAsync

### DISCUSSION THREAD

**Discussion by xin-peng** [microsoft/AirSim#4849] [Support Q&A]
Hello all,
I am a new user of airsim library. When I use the airsim moveToPositionAsync(x,y,z,velocity) api, the input velocity is referring to the final speed when drone reach (x,y,z), or the average speed for drone travel from current position to (x,y,z)? I feel this difference will be obvious when drone start with zero speed. 
Thank you!

**Comment by MehmetCagriK** [comment]
This won't be a direct answer but AirSim project is forked and actively supported by Codex Labs Inc. under [Colosseum](https://github.com/CodexLabsLLC/Colosseum)
